### DISCUSSION TITLE
Who has the longest uptime?

### DISCUSSION THREAD

**Discussion by adriel** [datarhei/restreamer#482] [General]
Just bit of a fun post, I just started using Restreamer, so pretty rookie numbers so far.

<img width="624" alt="image" src="https://user-images.githubusercontent.com/154217/205759116-dc483a84-97de-4805-a620-96976458962e.png">

**Share a screenshot of where your at.**
Ideally with the bitrate and FPS, I think it's interesting to see what other people's setup is like.

**Also, if it's not against any rules, share a link to what you're streaming.** (e.g. YouTube Live / Twitch etc)
Unfortunately my feed is of a driveway, so not very interesting, and don't really want to share that.

**Comment by svenerbeck** [comment]
Good idea. [electronic high5]

I still have an active older Docker container that needs an update to the latest version. It is for me to dev/test and has no output. The input is MTV Europe. 

![Bildschirm­foto 2022-12-06 um 11 29 07](https://user-images.githubusercontent.com/12124066/205887181-21d90038-27cd-469f-be02-65c98f3e1de2.png)

**Comment by Assiente** [comment]
I have some discrepancies in the uptime figures when looking at the built-in tools and the third-party service [https://www.host-tracker.com/en/ic/check-http](https://www.host-tracker.com/en/ic/check-http) But, in general, everything is within the permissible norm.

**Comment by ioppermann** [comment]
@Assiente Do you mind sharing some more details about the discrepancies?
